src/harness/sas.py

```diff
@@ -198,6 +215,11 @@ def InjectPalDatabaseRecord(self, request):
                  self._GetDefaultAdminSSLCertPath(),
                  self._GetDefaultAdminSSLKeyPath())
 
+  def InjectClusterList(self, request):
```

Comment:
Can you merge this branch to master to have the latest changes, it seems that this branch has some old changes?